src/ucp/core/ucp_context.c

```diff
@@ -287,6 +287,9 @@ static ucp_tl_alias_t ucp_tl_aliases[] = {
   { "ugni",  { "ugni_smsg", "ugni_udt:aux", "ugni_rdma", NULL } },
   { "cuda",  { "cuda_copy", "cuda_ipc", "gdr_copy", NULL } },
   { "rocm",  { "rocm_copy", "rocm_ipc", "rocm_gdr", NULL } },
+  { "gpu",   { "cuda_copy", "cuda_ipc", "gdr_copy", /* CUDA */
```

Comment:
given the below usage, not worth adding this alias

Reply:
removed